source/documentation/other-topics/Cronjobs.md

```diff
@@ -0,0 +1,56 @@
+### Kubernetes Cronjobs
+
+#### Overview
+
+Kubernetes has the concept of [Cronjobs][kubernetes-cronjobs] which creates Jobs on a time-based schedule, you can use CronJobs to run tasks at a specific time or interval. CronJobs are a good choice for automatic tasks, such as backups, reporting, sending emails, or cleanup tasks. 
+
+#### Usage
+
+CronJobs use [Job][kubernetes-jobs] objects to complete their tasks. A CronJob creates a Job object each time it runs. CronJobs are created, managed, scaled, and deleted in the same way as Jobs. Cron jobs require a config file, follow the [guide][cron-jobs] on creating a Cronjob. 
+
+This example cron job below prints the current time and a hello message every first minute of every hour from 0200 to 1400 UTC on Sunday, Monday, Friday, and Saturday
+
+example:
+
+```
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: hello
+spec:
+  schedule: "1 2-14 * * 0-1,5-6"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: hello
+            image: busybox
+            args:
+            - /bin/sh
+            - -c
+            - date; echo "Hello, World!"
+          restartPolicy: OnFailure
+```
+
+#### Applying Cronjob in your Namespace
+
+Make corresponding changes to [cronjob-ecr.yaml][cronjob-yaml] file and apply the Cronjob to your namespace in the kubernetes cluster, this runs a job periodically on a given schedule to delete untagged images in the ecr-repo, this will help to limit the count of Images in the ecr-repo.
```

Comment:
I think we should make it clear that this is an example file that people can copy and customise, or they can just create their own. Something like this:

> You need a yaml file to define your cronjob. You can either create your own, or use [this one][cronjob-yaml] as an example. This runs a job periodically...